DllImportGenerator/DllImportGenerator/TypeSymbolExtensions.cs

```diff
@@ -43,6 +41,18 @@ public static bool HasOnlyBlittableFields(this ITypeSymbol type)
             return true;
         }
 
+        private static bool GenericParameterContributesToBlittability(ITypeParameterSymbol typeParameter)
+        {
+            foreach (var attr in typeParameter.GetAttributes())
+            {
+                if (attr.AttributeClass?.Name == "ContributesToBlittabilityAttribute")
```

Comment:
I really dislike the `?` operator. I appreciate people like it, but try to avoid it when possible. Will `AttributeClass` ever be `null`? If so, when would that occur from a collection returned by `GetAttributes()`?